package.json

```diff
@@ -63,6 +64,7 @@
     "react-hook-form": "^7.53.0",
     "react-is": "^18.3.1",
     "react-merge-refs": "^2.1.1",
+    "react-remove-scroll": "2.5.5",
```

Comment:
Had to add these because we're patching them, and regenerating the lockfile would otherwise upgrade them.